sources/sections-a1-en/13-13th-cgpm.adoc

```diff
@@ -123,8 +123,8 @@ The 13th Conférence Générale des Poids et Mesures (CGPM),
 |===
 | wave number | 1 per metre | stem:["unitsml(m^(-1))"]
 | entropy | joule per kelvin | stem:["unitsml(J//K)"]
-| specific heat capacity | joule per kilogram kelvin | stem:["unitsml(J//kg*K)")] (((heat capacity)))(((joule (stem:["unitsml(J)"]))))
-| thermal conductivity | watt per metre kelvin | stem:["unitsml(W//m*K)")]
+| specific heat capacity | joule per kilogram kelvin | stem:["unitsml(J)"//("unitsml(kg)"*"unitsml(K)")] (((heat capacity)))(((joule (stem:["unitsml(J)"]))))
```

Comment:
I don't understand why you are doing this; is it for rendering? "unitsml(J//kg*K)" is a single unit. "unitsml(J)"//("unitsml(kg)"*"unitsml(K)" is indeed an operation involving three units, but the semantic intent is of a single unit generated from those three units.

Reply:
I just wanted to match with the original format, which I had understood, was the desiderata.
But yes, I agree with your argument. Is there a way to include the `cdot` in the units?